man/coredump.conf.xml

```diff
@@ -119,6 +119,22 @@
         either value to 0 to turn off size-based
         clean-up.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+         <term><varname>RateLimitInterval=</varname></term>
+         <term><varname>RateLimitBurst=</varname></term>
+         <listitem><para>Configures the rate-limiting that is applied to
+         core dump generated per daemon. Whenever a daemon crashes need to
```

Comment:
this is misleading, it's not "per daemon". that would suggest this was a per cgroup thing. But it isn't really. it's a per-comm-string thing, which is very different, from our usual definition of a daemon/service as a group of processes.

With my suggestion below to involve the exe name of the process I'd call this "per binary" I think.

Reply:
I have addressed the review comment. Please check once